docs/prometheus-scale.md

```diff
@@ -0,0 +1,108 @@
+Public ssh key file for Ansible.
+
+### PRIVATE_KEY
+Default: `~/.ssh/id_rsa`  
+Private ssh key file for Ansible.
+
+### ORCHESTRATION_USER
+Default: `root`  
+User for Ansible to log in as. Must authenticate with PUBLIC_KEY/PRIVATE_KEY.
+
+### WORKLOAD_IMAGE
+Default: `quay.io/openshift-scale/scale-ci-workload`  
+Container image that runs the workload script.
+
+### WORKLOAD_JOB_NODE_SELECTOR
+Default: `false`  
+Enables/disables the node selector that places the workload job on the `workload` node.
+
+### WORKLOAD_JOB_TAINT
+Default: `false`  
+Enables/disables the toleration on the workload job to permit the `workload` taint.
+
+### WORKLOAD_JOB_PRIVILEGED
+Default: `false`  
+Enables/disables running the workload pod as privileged.
+
+### KUBECONFIG_FILE
+Default: `~/.kube/config`  
+Location of kubeconfig on orchestration host.
+
+### PBENCH_INSTRUMENTATION
+Default: `false`  
+Enables/disables running the workload wrapped by pbench-user-benchmark. When enabled, pbench agents can then be enabled (`ENABLE_PBENCH_AGENTS`) for further instrumentation data and pbench-copy-results can be enabled (`ENABLE_PBENCH_COPY`) to export captured data for further analysis.
+
+### ENABLE_PBENCH_AGENTS
+Default: `false`  
+Enables/disables the collection of pbench data on the pbench agent Pods. These Pods are deployed by the tooling playbook.
+
+### ENABLE_PBENCH_COPY
+Default: `false`  
+Enables/disables the copying of pbench data to a remote results server for further analysis.
+
+### PBENCH_SSH_PRIVATE_KEY_FILE
+Default: `~/.ssh/id_rsa`  
+Location of ssh private key to authenticate to the pbench results server.
+
+### PBENCH_SSH_PUBLIC_KEY_FILE
+Default: `~/.ssh/id_rsa.pub`  
+Location of the ssh public key to authenticate to the pbench results server.
+
+### PBENCH_SERVER
+Default: There is no public default.  
+DNS address of the pbench results server.
+
+### SCALE_CI_RESULTS_TOKEN
+Default: There is no public default.  
+Future use for pbench and prometheus scraper to place results into git repo that holds results data.
+
+### JOB_COMPLETION_POLL_ATTEMPTS
+Default: `360`  
+Number of retries for Ansible to poll if the workload job has completed. Poll attempts delay 10s between polls with some additional time taken for each polling action depending on the orchestration host setup.
+
+### PROMETHEUS_CONCURRENCY
+Default: `10`
+Number of simultaneous request threads used by the prometheus loader tool.
+
+### PROMETHEUS_DURATION
+Default: `9000`
+The time in seconds that the prometheus loader should run for.
+
+### PROMETHEUS_GRAPH_PERIOD
+Default: `15`
+Prometheus query resolution in seconds.
+
+### PROMETHEUS_REFRESH_INTERVAL
+Default: `20`
+Sleep interval for each block iteration in seconds.
+
+### PROMETHEUS_SCALE_CLEANUP
+Default: `true`
+UNUSED.
+
+### PROMETHEUS_SCALE_TEST_PREFIX
+Default: `prometheus-scale`
+UNUSED.
+
+### PROMETHEUS_TEST_NAME
+Default: `testname01`
+Sets the name of the test directory for pbench.
```

Comment:
Can we add the smoke test variables for the prometheus workload, for example https://github.com/openshift-scale/workloads/blob/master/docs/nodevertical.md#smoke-test-variables.

Reply:
All the defaults are already set, you don't need to change/set any variables to just run the Prometheus job.

Reply:
prometheus_duration is set to 9000 by default, the smoke test variables are used just to check if the job is functional in case anyone wants to quickly run the test. We can set the prometheus_duration to 360 or something under the smoke section.

Reply:
Sure